http/backup_service.go

```diff
@@ -106,6 +131,73 @@ func (h *BackupHandler) handleBackupShard(w http.ResponseWriter, r *http.Request
 	}
 }
 
+func (h *BackupHandler) handleBackupMetadata(w http.ResponseWriter, r *http.Request) {
+	span, r := tracing.ExtractFromHTTPRequest(r, "BackupHandler.handleBackupMetadata")
+	defer span.Finish()
+
+	ctx := r.Context()
+
+	// Lock the sqlite and bolt databases prior to writing the response to prevent
+	// data inconsistencies.
+	h.BackupService.LockKVStore()
+	defer h.BackupService.UnlockKVStore()
+
+	h.SqlBackupRestoreService.LockSqlStore()
+	defer h.SqlBackupRestoreService.UnlockSqlStore()
+
+	baseName := time.Now().UTC().Format(influxdb.BackupFilenamePattern)
+
+	dataWriter := multipart.NewWriter(w)
+	w.Header().Set("Content-Type", "multipart/mixed; boundary="+dataWriter.Boundary())
+
+	parts := []struct {
+		fieldname string
+		filename  string
+		writeFn   func(io.Writer) error
+	}{
+		{
+			"kv",
+			fmt.Sprintf("%s.bolt", baseName),
+			func(fw io.Writer) error {
+				return h.BackupService.BackupKVStore(ctx, fw)
+			},
+		},
+		{
+			"sql",
+			fmt.Sprintf("%s.sqlite", baseName),
+			func(fw io.Writer) error {
+				return h.SqlBackupRestoreService.BackupSqlStore(ctx, fw)
+			},
+		},
+		{
+			"buckets",
```

Comment:
Update this part to be named "buckets". It will have a file name with a `.json` extension.